src/api/connectors.ts

```diff
@@ -71,6 +63,8 @@ const DETAILS_FORM_QUERY = `
     )
 `;
 
+// TODO: Remove getConnectors_detailsForm and related assets.
+//   It is only used by the test JSON forms page.
```

Comment:
Please rename this function to make it clear it is just for the test page.

Reply:
Sure. However, using that logic, one could argue that the details from store `hydrateState` action and every related helper function should be renamed. Altering function names is a little overkill as the test page should be updated in a fast follow to use the hydrator hook, but I will update the function called out since it was requested.